### ISSUE TITLE
Press and Hold Action No Longer Brings Up Options

### ISSUE THREAD

**Issue by Boh1** [sstratoti/actionable-notifications-subflow-for-ios#5]
I've been happily using this subflow for some time now and it used to work just great. But now, when an alert is displayed, while I get the usual alert + "Press and Hold for More Options" text that I use, when pressing and holding all I see is the same message, without the options being displayed. 

I do get haptic feedback from my press and hold, but the options are never displayed. This is not how this subflow originally worked and I don't know if it's some form of version mismatch between this subflow, Home Assistant (2023.4.6) or the HA mobile app (2023.4) or something that's changed in iOS (16.4.1).

I really liked this when it was working - can you offer any debug advice?

**Comment by sstratoti** [comment]
Hmm, it's working for me on 16.4.1 with 2023.4.4. I'll try upgrading tonight to see if anything changed. I need to load up and updated version of the script anyways.

**Comment by sstratoti** [comment]
I ran my unit tests again and noticed an error with long/lat being populated incorrectly. Aside from that, I think I made some updates back in Dec/Jan that I hadn't pushed up. It might correct your issue.

Please try the latest version? I've just updated the flow. Let me know if you're still having this issue. Thanks!

**Comment by sstratoti** [comment]
I've added a new "Debug Mode?" Boolean to the flow. It's all the way at the bottom of the config for the node. Can you switch that to "checked" / true, and then open your debug window in node red? 

It should output several Debug messages when calling the iOS node (if set to true).

There will be up to 4 of them, depending if you click an action / clear the message. They should all start with "iOS Notify".

Example:

<img width="703" alt="image" src="https://user-images.githubusercontent.com/1392597/235319207-62dccf5a-0028-4e07-b663-5b82ab1d30c2.png">

Can you copy those values (like in the screenshot above) and add them to this issue? It'll help me debug what is wrong in the service call JSON.

Thank you!

**Comment by Boh1** [comment]
Okay, I installed the new flow and configured it similar to another notification that was exhibiting the behavior noted in my original post. I still get the same problem: press and hold brings up no options. Double tapping on the notification just opens the HA app on my phone. 

Here's a screenshot of some of the config I used for the subflow:

<img width="470" alt="config" src="https://user-images.githubusercontent.com/16505721/235323545-53071553-174a-41a3-b138-bfbb6bf17029.png">

And here's the debug output as requested (only two messages):

<img width="225" alt="debug_1" src="https://user-images.githubusercontent.com/16505721/235323818-c5058c2a-8358-42b7-ba63-822e09dd1944.png">


<img width="224" alt="debug_2" src="https://user-images.githubusercontent.com/16505721/235323821-b85d6a5a-0990-4760-8b22-ae469e2d3080.png">

I also tested the same flow with my iPad and had exactly the same result. Also checked with th HA app connected via an (internal) LAN address versus my (external) CloudFlare tunnel. Again, same result.

Thanks for your help - just let me know if there's any more tests that you would like me to run.
